lib/stack_master/parameter_resolvers/ami_finder.rb

```diff
@@ -0,0 +1,36 @@
+module StackMaster
+  module ParameterResolvers
+    class AmiFinder
+      def initialize(region)
+        @region = region
+        @owner = ['self']
+      end
+
+      def build_filters(value, prefix = nil)
+        owner_id = []
+        filters = value.split(',').map do |name_with_value|
+          name, value = name_with_value.strip.split('=')
+          owner_id.push value if name == 'owner_id'
+          name = prefix ? "#{prefix}:#{name}" : name
+          { name: name, values: [value] }
+        end
+        @owner = owner_id unless owner_id.empty?
+        filters
+      end
```

Comment:
It'd be better to name the variables `@owners` and `owner_ids` since we're dealing with arrays and the first one then matches up with the api call a bit better.